views/pod.slim

```diff
@@ -103,10 +103,6 @@ ruby:
 
       hr
 
-    - if is_binary
```

Comment:
Is this intentional?

Reply:
Yeah, I'd been wanting to remove that for a while. 

Code stripping and bitcode etc makes these numbers inaccurate enough on what it looks like when integrated that I don't feel comfortable letting people feel like they can use this as a metric